src/redux/rootReducer.ts

```diff
@@ -79,6 +92,15 @@ export const reducers = {
   deliverablesEdit: deliverablesEditReducer,
   deliverablesSearch: deliverablesSearchReducer,
   deliverables: deliverablesReducer,
+  documentProducerDocumentList: documentListReducer,
```

Comment:
Would it be time to export single payloads of reducers from the xyzslice files? We could import the payload and spread that in the rootReducer.
I can take a quick stab at it.

Reply:
Yea that would be cleaner, I can do that now.